src/main/java/frc/robot/commands/RelinquishCommand.java

```diff
@@ -0,0 +1,43 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Acquisition;
+
+public class RelinquishCommand extends Command {
```

Comment:
Is this the command that gets rid of the note without scoring it? just calling it relinquish is very vague. I would encourage you to rename this class and be more descriptive